pages/linux/ifdown.md

```diff
@@ -0,0 +1,11 @@
+# ifdown
+
+> Tool used to disable network interfaces.
```

Comment:
Yes, you make a good point -- I actually had the same reservations when I wrote my suggestion. Let's keep "disable" then.